### ISSUE TITLE
Enable elements hover effects to be trigger by other elements/class/id 

### ISSUE THREAD

**Issue by miguel-cravidao** [elementor/elementor#8848]
Hello,

I think that one of the next steps for a major jump on web design with Elementor would be hover effects based on other elements instead only the element itself.

For instance, if I hover on a column (first element) some text inside of it (second element) changes its colour.
So in this example, I would select the text element, then the hover effects tab I would have a selector to let me choose/type an element type/class/id that will be the trigger for this hover effect.

At the moment we can only configure hover effects for the element itself with no relationship to the other UI elements, dramatically shortening the UI interactivity.

**Comment by zahinwadud** [comment]
+1 for this feature.  And it would be great with clickable column feature. 
Currently this feature is available in CTA widget. If the CTA widget is hovered then the image can show hover effect. But the thing is CTA is very limited as per adding elements.

**Comment by R98rayan** [comment]
yes we need this as soon as possible :D

**Comment by octopus4056** [comment]
Agreed! This is critical! Especially without the ability to define custom CSS properties or use CSS variables, there seems to be no way to make this happen on an Elementor site. Not acceptable. Modern web design is interactive and we need the ability (at minimum via CSS) to structure complex interactions between elements. Please prioritize! Thanks!
